pyproject.toml

```diff
@@ -60,3 +60,14 @@ extend-select = [
 docstring-quotes = "double"
 inline-quotes = "single"
 multiline-quotes = "single"
+
+[tool.uv]
+dev-dependencies = [
+    "pytest>=6.0",
```

Comment:
Open question: do we have to freeze these to the specific versions, like in `requirements.txt` right now?